pontoon/base/models.py

```diff
@@ -540,6 +541,11 @@ class Locale(AggregatedStats):
         """
     )
 
+    transvision = models.BooleanField(default=False, help_text="""
```

Comment:
btw. Did you consider to add opt-in/opt-out for user profile for all systems supported in machinery tab?

Reply:
No. Why would we *not* serve suggestions from some source to some people?

Reply:
I just thought that some users may prefer some systems over the other.

Reply:
We haven't gotten any requests for this.